Library/Homebrew/dev-cmd/unpack.rb

```diff
@@ -44,34 +59,71 @@ def run
 
         odie "Cannot write to #{unpack_dir}" unless unpack_dir.writable?
 
-        formulae.each do |f|
-          stage_dir = unpack_dir/"#{f.name}-#{f.version}"
-
-          if stage_dir.exist?
-            odie "Destination #{stage_dir} already exists!" unless args.force?
+        formulae_and_casks.each do |formula_or_cask|
+          next if formula_or_cask.is_a?(Keg)
```

Comment:
`args.named.to_formulae_and_casks` can return Kegs, which I think is ok to discard here, please correct if I'm wrong!